packages/core/src/App/AppContent.tsx

```diff
@@ -44,7 +55,18 @@ const AppContent: React.FC<{ passthrough: unknown }> = observer(({ passthrough }
     const { isMobile } = useDevice();
     const { switchLanguage } = useTranslations();
 
-    const { isOAuth2Enabled } = useOauth2();
+    const { isOAuth2Enabled, oAuthLogout } = useOauth2({
+        handleLogout: async () => {
+            await logout();
+        },
+    });
+
+    useSilentLoginAndLogout({
```

Comment:
This hook is to check the logged_state cookie for silent login and single logout